### PR TITLE
Big table

### PR DESCRIPTION
**Pull request by PMarcL**

### CODE REVIEW

File: scss/tables/big-table.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  tbody {
+    tr {
+      height: $big-table-height;
+
+      cursor: default;
+
+      td {
+        padding: $table-cell-left-right-padding;
+
+        vertical-align: top;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: Is this really necessary?

File: scss/tables/big-table.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+      td.main {
+        white-space: nowrap;
+
+        .icon {
+          width: $big-table-height;
+          height: $big-table-height;
+          margin-right: 20px;
+
+          vertical-align: -3px;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: Can we do something better here ?

File: scss/tables/big-table.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        vertical-align: top;
+      }
+
+      td.main {
+        white-space: nowrap;
+
+        .icon {
+          width: $big-table-height;
+          height: $big-table-height;
+          margin-right: 20px;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: I would use a variable here.

File: scss/tables/big-table.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+          vertical-align: -3px;
+        }
+
+        div.details {
+          display: inline-block;
+
+          div.second-row {
+            .icon {
+              width: 15px;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: Missing variables for the icon size.

File: scss/tables/big-table.scss [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            }
+          }
+        }
+      }
+
+      td.col, td.main {
+        position: relative;
+
+        .first-row {
+          font-size: 15px;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: You should use a variable here for the font-size and the line height.

File: scss/tables/big-table.scss [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+          line-height: 20px;
+        }
+
+        div.second-row {
+          margin-top: 2px;
+
+          font-size: 12px;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: Same here, use variables.

File: scss/tables/big-table.scss [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+      td.col, td.main {
+        position: relative;
+
+        .first-row {
+          font-size: 15px;
+          line-height: 20px;
+        }
+
+        div.second-row {
+          margin-top: 2px;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: Should add a comment (and or variable)

File: scss/tables/big-table.scss [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+          font-size: 12px;
+          line-height: 17px;
+        }
+      }
+
+      div.wrapper {
+        min-height: $big-table-height;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: big-table-height ? big-table-row-height would sound better to me.

File: scss/tables/big-table.scss [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+      }
+
+      &.disabled td .wrapper {
+        opacity: 0.5;
+      }
+    }
+
+    tr.error-blankslate-row {
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: I don't think this should be in the `big-table.scss` file, but that's only my opinion. Your choice.

File: scss/tables/big-table.scss [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+      &:hover {
+        background-color: $pure-white;
+      }
+
+      td.error-blankslate-section {
+        padding: $big-table-blankslate-section-padding;
```
Comment (JPLachance) [inline scss/tables/big-table.scss]: In my opinion, a blank slate in a normal table or a big table is the same thing. I think we are too specific here.

**Comment by JPLachance** [comment]
And, I would love a "big table" example in the Jekyll doc :)

**Comment by PMarcL** [comment]
Yeah, I thought about it, but setting up ruby and jekyll on pc is a nightmare and it didn't work on my work laptop... I'll try with my personnal laptop.

**Comment by wfortin** [comment]
Yeah Jekyll is a pain on windows. Might want to check https://github.com/coveo/source.coveo.com#run-locally for a docker version of jekyll.

**Pull request merged**